### PR TITLE
Add options to choose specific K.K. songs

### PR DESCRIPTION
**Pull request by MaxSchaefer**
Issue: https://github.com/animal-crossing-music-extension/ac-music-extension/issues/56

_#hacktoberfest2020_

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -133,6 +134,13 @@ <h4>Version Choice</h4>
 							<input id="kk-version-both" name="kk-version" type="radio"><label for="kk-version-both"><span><span></span></span>Both</label>
 						</div>
 					</section>
+					<section>
+						<h4>Select songs</h4>
+						<div class="checkbox">
+							<input id="kk-songs-selection-enable" type="checkbox"><label for="kk-songs-selection-enable"><span></span>Only play the selected K.K. songs</label>
+						</div>
+						<select id="kk-songs-selection" style="height: 25vh" multiple></select>
```
Comment (PikaDude) [inline options.html:142]: Maybe add some sort of indication to use the CTRL key to select multiple songs?

**Review by PikaDude — APPROVED** [review]
Looks good! I only have one small suggestion - if possible.

File: options.html (hunk 1 above)
Comment (MaxSchaefer) [reply options.html:142]: I added a simple hint text

**Comment by paladique** [comment]
@MaxRobinSchaefer thank you for your contribution!! 🐾

**Pull request merged**